foodhub_picking_operations/reports/report_picking_operations.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="report_picking" inherit_id="stock.report_picking">
+        <xpath expr="//h1" position="replace">
+            <h1 t-field="o.origin" class="mt0"/>
+        </xpath>
```

Comment:
avoid using `replace` as much as possible, to improve compatibility with other modules and to avoid unnecessary copying of other attributes.

```suggestion
        <xpath expr="//h1" position="attributes">
            <attribute name="t-field">o.origin</attribute>
        </xpath>
```